doc/articles/features/native-frame-nav.md

```diff
@@ -0,0 +1,48 @@
+# Native Frame Navigation
+
+## Introduction
+Since Uno.UI is written under the UWP api contract, most of the how-tos are sharable. In this article, we will discuss the particularities of the native `Frame` navigation in detail.
+
+It is recommended to be familiar with the concept of navigation in UWP, here are some good starting points:
+- [Navigation design basics for Windows apps](https://docs.microsoft.com/en-us/windows/uwp/design/basics/navigation-basics)
+- [`Windows.UI.Xaml.Controls.Frame`](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.Frame)
+
+## Native Frame Navigation
+On Android and iOS, there is the option to enable the native frame navigation which replaces below controls' UWP style & template by their native counterpart:
+- `Windows.UI.Xaml.Controls.Frame`<superscript>*</superscript>
+- `Windows.UI.Xaml.Controls.CommandBar`
+- `Windows.UI.Xaml.Controls.AppBarButton`
+
+This feature can be enabled by calling the method below typically in the constructor of `App` class:
+```cs
+#if __IOS__ || __ANDROID__
+    Uno.UI.FeatureConfiguration.Style.ConfigureNativeFrameNavigation();
+#endif
+```
+> note: It is recommended to enable this feature in order to provide a more native user experience.
```

Comment:
```suggestion
> [!NOTE]
> It is recommended to enable this feature in order to provide a more native user experience.
```

(This is docfx syntax to add a note block: https://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html#note-warningtipimportant )